src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
It is a good practice to add `box-sizing: border-box;` on all elements `* {...}`